doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_fetch.rst

```diff
@@ -112,15 +110,13 @@ To reconfigure the sample to use the fetch model, complete the following steps:
          - suit-directive-fetch:
            - suit-send-record-failure
 
-   #. Modify the ``suit-install`` sequence to use an identical URI, (as in the ``suit-payload-fetch``), instead of the integrated one:
+  #. Modify the ``suit-install`` sequence to use an identical URI, (as in the ``suit-payload-fetch``), instead of the integrated one:
 
      The SUIT procedure attempts to use all fetch sources registered with :c:func:`suit_dfu_fetch_source_register` until one of them fetches the payload.
      If no sources are able to fetch the payload, the update process ends with an error.
 
      The reference SMP fetch source implementation only recognizes URIs that start with ``file://``.
 
-   #. Modify the ``suit-install`` sequence to use an identical URI, as in the ``suit-payload-fetch``, instead of the integrated one.
```

Comment:
The `#. Modify the ``suit-install`` sequence...`
should come after `The reference SMP fetch source...`